<launch>
    <!--<node pkg="map_server" type="map_server" name="map_server_node" args="$(find static_map)/map_file/map.yaml"/>-->
    <node pkg="rviz" type="rviz" name="Rviz_for_extract_path" args="-d $(find visualization_module)/rviz/rviz.rviz" />
    <node pkg="visualization_module" type="dynamic_visualization" name="visualization" output="screen">
        <param name="map_file" value="$(find static_map)/map_file/yuquan.pcd"/>
        <param name="offset_x" value="5.0"/>
        <param name="offset_y" value="-5.0"/>
        <param name="offset_z" value="5.0"/>
    </node>
    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="reconfiguration" />
</launch>
